docs/guidelines/index.md

```diff
@@ -0,0 +1,176 @@
+
+To get started, join our Slack channel and introduce yourself.
+
+In the first communication, we would be interested to see the following information:
+
+- A short self-introduction: your area of study, interests, background
+- Why are you interested in the Fosslogy project? Which projects ideas do you want to work on?
+- If you participated in open-source projects, please reference them
+- If you have a GitHub, Twitter account, blog, or any technical/scientific publications, don't hesitate to share those as well..
+
+As you dive into the codebase, use the Slack channel to ask for feedback on your project ideas and for help as you navigate the code. 
+
+Remember, the more specific and clear your questions are, the more likely you are to get a helpful response. So don't be afraid to get specific, like: ``I'm interested in Project X, and I've noticed issues Y and Z. It looks like a, b, and c are already implemented. Can I pitch a project that would complete d, e, and f?"
+We're here to help you make the most of your time with us, so don't hesitate to reach out!``
+
+## First contribution
+
+Looking to make your mark in the world of open-source and increase your chances of being selected for Google Summer of Code? The key is to dive in and get your hands dirty with the software.
+
+What better way to learn about a tool than by contributing to its development? Whether fixing bugs or adding new features, your contributions can make a real difference.
+
+Here's a general roadmap to follow:
+
+- Find an issue, bug, or proposed feature that catches your interest.
+- Make a copy of the repository (known as "forking").
+- Roll up your sleeves and make your improvements.
+- Submit a well-documented pull request, including all the relevant details about your changes.
+- Be patient and wait for feedback and discussion on Github (reviews can take some time).
+
+We can't stress enough the importance of making some contributions to the project before you submit your GSoC application. Not only will it help you fine-tune your proposal, but it will also show the mentors that you're invested in the community and are dedicated to making a difference.
+
+### Useful links
+
+Here is a list of links to help you get started on participating in Fosslogy and in coding for the FOSSology project.
+
+[Install from Source](https://github.com/fossology/fossology/wiki/Install-from-Source)
+
+[Using FOSSology](https://github.com/fossology/fossology/wiki#using-fossology)
+
+[Contribute to FOSSology](https://github.com/fossology/fossology/wiki#get-started-with-development)
+
+[Get Started with Development](https://github.com/fossology/fossology/wiki#get-started-with-development)
+
+[FOSSology Agents](https://github.com/fossology/fossology/wiki/FOSSology-Agents)
+
+There is also a [list of newbie-friendly issues](https://github.com/fossology/fossology/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```

Comment:
How about using this link: https://github.com/fossology/fossology/contribute ??

Reply:
Yes sir, we can use.